Automattic/simplenote-macos
# Shortcuts: Find note with tag

### Fix
So one thing users might want to do is to access their notes in shortcuts by seeing what notes are part of different tags.  This PR allows for that.

The shortcut intent will allow users to pick a tag, then simplenote will return all the notes that tag is associated with, and the user can choose between them.

On it's own this shortcut is limited, but connect it to Open Note or Copy Note Content and it becomes quite useful

### Test
1. Run the branch and then open the shortcuts app. Create a new shortcut and look for Simplenote in the apps
2. Find Note with Tag.  Select a tag that you know has notes connected to it
3. Now select Open note from the sidebar.
4. run the shortcut.
5.  Confirm that the note you choose is opened in simplenote

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
